airflow/providers/google/provider.yaml

```diff
@@ -193,6 +191,9 @@ additional-extras:
     dependencies:
       - apache-airflow-providers-amazon>=2.6.0
 
+devel-dependencies:
```

Comment:
In 3.49.1 there is no 'testing' extra any more - because google team decided to make it back a "requirement" - so we cannot really get this one :(